pkg/virt-config/deprecation/feature-gates.go

```diff
@@ -76,6 +78,7 @@ func init() {
 
 	RegisterFeatureGate(FeatureGate{Name: PasstGate, State: Discontinued, Message: PasstDiscontinueMessage, VmiSpecUsed: passtApiUsed})
 	RegisterFeatureGate(FeatureGate{Name: MacvtapGate, State: Discontinued, Message: MacvtapDiscontinueMessage, VmiSpecUsed: macvtapApiUsed})
+	RegisterFeatureGate(FeatureGate{Name: DockerSELinuxMCSWorkaround, State: Deprecated, Message: "DockerSELinuxMCSWorkaround has been discontinued since v1.4. As an alternative, please add the annotation kubevirt.io/DockerSELinuxMCSWorkaround to kubevirt CR."})
```

Comment:
Please update the code here to use the new annotation name, maybe directly use the v1 api package with Sprintf to avoid this (if you don't have to add any deps).

Reply:
Done.